docs/rest_api.md

```diff
@@ -246,13 +246,16 @@ Returns a JSON dictionary of dictionaries, where the exterior dictionary is inde
 the network name and the value of each key is another dictionary with keys corresponding
 to that network's id and projects
 
+Response contains all networks if having the administrative access. Otherwise, response
```

Comment:
I'd say something like `Response contains all networks if the user is an admin.` ...

Reply:
I think this grammar fix is important, but otherwise I'm happy.